css-two-column-layout/styles.css

```diff
@@ -0,0 +1,95 @@
+* {
+  font-family: karla, monospace;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 1000px;
+}
+
+.main-header,
+.sub-header,
+button {
+  text-transform: uppercase;
+}
+
+.main-header {
+  flex-basis: 100%;
+  text-align: center;
+  margin-top: 20px;
+  font-size: 150%;
+  font-weight: bold;
+  letter-spacing: 0.1rem;
+}
+
+.main-header-description {
+  flex-basis: 70%;
+  text-align: center;
+  margin-top: -30px;
```

Comment:
In general negative margins should be avoided. If you feel that you need a negative margin, that usually just means that an adjacent element's margin should be reduced.